spec/model/trial_user_spec.rb

```diff
@@ -0,0 +1,28 @@
+require 'rails_helper'
+
+RSpec.describe 'User', type: :model do
+  it 'is valid with valid attributes' do
+    user1 = User.new(email: 'email@example.com', password: 'password')
+    expect(user1).not_to be_valid
+  end
+
+  it 'is not valid without first_name' do
+    user2 = User.new(first_name: nil)
+    expect(user2).not_to be_valid
+  end
+
+  it 'is not valid without last_name' do
+    user2 = User.new(last_name: nil)
+    expect(user2).not_to be_valid
+  end
+
+  it 'is not valid without email' do
+    user2 = User.new(email: nil)
+    expect(user2).not_to be_valid
+  end
+
+  it 'is not valid without username' do
+    user2 = User.new(username: nil)
+    expect(user2).not_to be_valid
+  end
```

Comment:
Be specific with your test. If you are testing the validation presence of attribute `last_name` then you should provide the values to the rest of the user's attributes except for `last_name`. Or you may better use this gem https://github.com/thoughtbot/shoulda